### PR TITLE
Fix postsign

### PR DESCRIPTION
**Pull request by gavofyork**
Yet to do:
- [x] fix tests;
- [x] fix "it thinks it's an client-side account even when it's not" bug;
- [x] detect ASCII data and provide link to render as text in popup window.

### CODE REVIEW

**Comment by gavofyork** [comment]
@tomusdrw any idea about the "it thinks it's an client-side account even when it's not" bug?

**Comment by tomusdrw** [comment]
I've experienced something similar today when switching networks without reloading the GUI, maybe that was the case?
Will check if it works fine for me in a second.

**Comment by tomusdrw** [comment]
No, it's a different problem:

Here:
https://github.com/ethcore/parity/blob/f627bef08b5eb47e59c14ff6f0308c7b07e72c0e/js/src/views/Signer/components/TransactionPendingForm/TransactionPendingFormConfirm/transactionPendingFormConfirm.js#L259

`address` is lower cased, whereas `accounts` have checksum-cased addresses. Should be easy to fix.

**Comment by tomusdrw** [comment]
Logged as separate issues: #4352 #4353

**Comment by gavofyork** [comment]
fixed both.

**Comment by gavofyork** [comment]
@tomusdrw ready for re-review.

File: js/src/views/Signer/components/SignRequest/signRequest.css [hunk 1]
```diff
@@ -27,8 +27,18 @@
   min-height: $pendingHeight;
 }
 
+.signData {
```
Comment (tomusdrw) [inline js/src/views/Signer/components/SignRequest/signRequest.css]: Might be good to introduce maximal height and overflow for very long ascii texts:
```
overflow: auto;
max-height: 6em;
```

File: js/src/views/Signer/components/SignRequest/signRequest.js [hunk 2]
```diff
@@ -62,8 +76,21 @@ export default class SignRequest extends Component {
     );
   }
 
+  renderAsciiDetails (ascii) {
+    return (<div className={ styles.signData }>
```
Comment (tomusdrw) [inline js/src/views/Signer/components/SignRequest/signRequest.js]: It's most common in the codebase to put a new line after `(` and before `)`.

**Pull request merged**